sycl/test-e2e/ESIMD/histogram_256_slm.cpp

```diff
@@ -207,5 +207,7 @@ int main() {
   else
     std::cout << "FAILED\n";
 
+  free(hist);
```

Comment:
Minor comment: 'hist' is allocated with 'new'[], not 'malloc', thus being pedantic it should be deallocated with 'delete[] hist;' not 'free(hist)'.

Reply:
Ok, free() may work for int* allocated with new[] as `int` type does not need calling destructor for every element

Reply:
Fixed.